scripts/busybox-ping-test/alcor_services.ini

```diff
@@ -16,43 +16,49 @@ network_acl_manager    = {"name":"nam",    "port":9013}
 network_config_manager = {"name":"ncm",    "port":9014}
 gateway_manager        = {"name":"gm",     "port":9015}
 
-
 [AlcorControlAgents]
 node1 = 10.213.43.161
 node2 = 10.213.43.163
 
-
 [test_setup]
```

Comment:
I was wondering whether the same configuration in [test_case1] or [test_case2] will overwrite those in [test_setup]. 

For example., "subnet_id" in test_setup and "subnet_ids" in test_case1.

Reply:
Yes, it will overwrite. We have the following breakup:
test_setup - for simple test case of two nodes pinging each other when on same subnet.
test_case1 - two nodes on different subnet but same security group
test_case2 - two nodes on same subnet but different security group.